### PR TITLE
fix(vsc): select workspace folder canceled in WSL

### PR DESCRIPTION
**Pull request by tecton**
Related bug: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/15772677

E2E TEST: N/A

### CODE REVIEW

File: packages/vscode-extension/src/qm/vsc_ui.ts [hunk 1]
```diff
@@ -504,9 +504,6 @@ export class VsCodeUI implements UserInteraction {
 
           disposables.push(
             quickPick.onDidAccept(onDidAccept),
-            quickPick.onDidHide(() => {
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/qm/vsc_ui.ts:507]: Is there any side effects when user press Esc in the quick pick?

Comment (tecton) [reply packages/vscode-extension/src/qm/vsc_ui.ts:507]: ESC can still cancel the quick pick. It invokes `onDidHide` as well.

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/src/qm/vsc_ui.ts:507]: When it invokes `onDidHide`, will it trigger UserCancelError and send correct telemetry?

Comment (tecton) [reply packages/vscode-extension/src/qm/vsc_ui.ts:507]: Yes, I'll update the code. Otherwise, the telemetry will be lost.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**